nmproxy/models/models.go

```diff
@@ -6,55 +6,29 @@ import (
 	"fmt"
 	"net"
 	"sync"
-	"time"
 
+	"github.com/gravitl/netmaker/models"
 	"golang.zx2c4.com/wireguard/wgctrl/wgtypes"
 )
 
-// ProxyAction - type for proxy action
-type ProxyAction string
-
 const (
 	// default proxy port
 	NmProxyPort = 51722
 	// default CIDR for proxy peers
 	DefaultCIDR = "127.0.0.1/8"
-	// PersistentKeepaliveInterval - default keepalive for wg peer
-	DefaultPersistentKeepaliveInterval = time.Duration(time.Second * 20)
-
-	// ProxyUpdate - constant for proxy update action
-	ProxyUpdate ProxyAction = "PROXY_UPDATE"
-	// ProxyDeletePeers - constant for proxy delete peers action
-	ProxyDeletePeers ProxyAction = "PROXY_DELETE"
-	// ProxyDeleteAllPeers - constant for proxy delete all peers action
-	ProxyDeleteAllPeers ProxyAction = "PROXY_DELETE_ALL"
-	// NoProxy - constant for no ProxyAction
-	NoProxy ProxyAction = "NO_PROXY"
 )
 
 // PeerConnMap - type for peer conn config map
 type PeerConnMap map[string]*Conn
 
-// Proxy - struct for proxy config
-type Proxy struct {
-	PeerPublicKey  wgtypes.Key
```

Comment:
I believe this only model is used only on netclient side, should be kept here

Reply:
moved back to netclient